src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/__tests__/HealthCertifiedPersonCellModelTests.swift

```diff
@@ -1059,9 +1063,9 @@ class HealthCertifiedPersonCellModelTests: XCTestCase {
 		)
 
 		XCTAssertEqual(cellModel.qrCodeViewModel.covPassCheckInfoPosition, .bottom)
-		XCTAssertFalse(cellModel.isStatusTitleVisible)
 		XCTAssertTrue(cellModel.switchableHealthCertificates.isEmpty)
-		XCTAssertNil(cellModel.shortStatus)
+		XCTAssertNil(cellModel.shortAdmissionStatus)
+		XCTAssertNil(cellModel.maskStatus)
```

Comment:
Can you add similar tests like `testWithoutAdmissionState()` and `testWithEmptyAdmissionState()` for the mask state? Couldn't find one for e.g. maskState.visible = false